core/services/keystore/keys/p2pkey/export.go

```diff
@@ -42,17 +40,13 @@ func (key KeyV2) ToEncryptedJSON(password string, scryptParams utils.ScryptParam
 		password,
 		scryptParams,
 		adulteratedPassword,
-		func(id string, key KeyV2, cryptoJSON keystore.CryptoJSON) (EncryptedP2PKeyExport, error) {
-			rawPubKey, err := key.GetPublic().Bytes()
-			if err != nil {
-				return EncryptedP2PKeyExport{}, errors.Wrapf(err, "could not get raw public key")
-			}
+		func(id string, key KeyV2, cryptoJSON keystore.CryptoJSON) EncryptedP2PKeyExport {
```

Comment:
This was the only instance actually returning an error, so we can simplify the signature now.